src/main/java/com/buschmais/jqassistant/neo4j/embedded/impl/Neo4jCommunityNeoServer.java

```diff
@@ -80,11 +83,19 @@ public void stop() {
         }
     }
 
+    /**
+     * @deprecated Replaced by Neo4j plugins mechanism.
+     */
+    @Deprecated
```

Comment:
Wasn't there an annotation to set the version in which a deprecated method will be removed? Seems like a good fit here.